### PR TITLE
fix(no-input-prefix): exact strings not being reported

### PR DESCRIPTION
**Pull request by rafaelss95**
Fixed according to discussion in #594.

### CODE REVIEW

File: test/noInputPrefixRule.spec.ts [hunk 1]
```diff
@@ -36,6 +36,25 @@ describe(ruleName, () => {
...
     });
 
+    it('should fail when an input property is strictly equal to a blacklisted prefix', () => {
+      const prefixes = ['should'];
+      const propertyName = prefixes[0];
+      const inputExpression = `@Input() ${propertyName}: boolean;`;
+      const source = `
+        @Directive()
+        class ${className} {
+          ${inputExpression}
```
Comment (mgechev) [inline test/noInputPrefixRule.spec.ts]: Let's not use interpolation that aggressively. If we duplicate `should` twice in a single test case, this is fine. Currently, the code is a bit hard to follow.

Comment (rafaelss95) [reply test/noInputPrefixRule.spec.ts]: Right. Changing...

Comment (rafaelss95) [reply test/noInputPrefixRule.spec.ts]: Done. I hope it's clear now :)
